generators/Exercises/Generators/SaddlePoints.cs

```diff
@@ -10,11 +10,10 @@ public class SaddlePoints : GeneratorExercise
     {
         protected override void UpdateTestMethod(TestMethod testMethod)
         {
-            testMethod.TestedMethodType = TestedMethodType.InstanceMethod;
+            testMethod.TestedMethodType = TestedMethodType.StaticMethod;
```

Comment:
I think you should be able to remove this line. Running the generator again should have the same outcome.